### PR TITLE
Llama v2 70 b lora update

### PR DESCRIPTION
**Pull request by itayhubara**

### CODE REVIEW

File: mlperf_logging/benchmark_meta.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
         'ncf': 10,
         'rnnt': 10,
         'unet3d': 40,
+        'llama2_70b_lora': 10,
```
Comment (mmarcinkiewicz) [inline mlperf_logging/benchmark_meta.py:18]: I'm not convinced this is a good benchmark name, seems a bit overcomplicated. Have we agreed on this? How about just `lora`?

File: mlperf_logging/compliance_checker/closed_common.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
```
Comment (mmarcinkiewicz) [inline mlperf_logging/compliance_checker/closed_common.yaml:1]: The whole file seems unnecessary here?

File: mlperf_logging/compliance_checker/mlp_compliance.py [hunk 3]
```diff
@@ -263,13 +263,13 @@ def check_loglines(self, loglines, config):
           self.put_message('No log lines detected')
 
         enqueue_config(config)
-
+
```
Comment (mmarcinkiewicz) [inline mlperf_logging/compliance_checker/mlp_compliance.py:266]: what happened here?

File: mlperf_logging/compliance_checker/training_4.0.0/closed_llama2_70b_lora.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+- KEY:
+    NAME:  opt_gradient_clip_norm
+    REQ:   EXACTLY_ONE
+
+- KEY:
+    NAME:  opt_adamw_weight_decay
+    REQ:   EXACTLY_ONE
+
+- KEY:
+    NAME:  gradient_accumulation_steps
```
Comment (mmarcinkiewicz) [inline mlperf_logging/compliance_checker/training_4.0.0/closed_llama2_70b_lora.yaml:28]: duplicated below

File: mlperf_logging/compliance_checker/training_4.0.0/closed_llama2_70b_lora.yaml [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    REQ:   EXACTLY_ONE
+
+- KEY:
+    NAME:  gradient_accumulation_steps
+    REQ:   EXACTLY_ONE
+
+- KEY:
+    NAME:  lora_alpha
```
Comment (mmarcinkiewicz) [inline mlperf_logging/compliance_checker/training_4.0.0/closed_llama2_70b_lora.yaml:36]: duplicated in L12

File: mlperf_logging/compliance_checker/training_4.0.0/closed_llama2_70b_lora.yaml [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    REQ:   EXACTLY_ONE
+
+- KEY:
+    NAME:  lora_rank
```
Comment (mmarcinkiewicz) [inline mlperf_logging/compliance_checker/training_4.0.0/closed_llama2_70b_lora.yaml:40]: How about we put constrains on this value to 16?

File: mlperf_logging/mllog/constants.py [hunk 7]
```diff
@@ -167,7 +167,7 @@
 EVAL_MAX_PREDICTION_SYMBOLS = "eval_max_prediction_symbols"
 START_WARMUP_STEP = "start_warmup_step"
 INIT_CHECKPOINT_STEP = "init_checkpoint_step"
-
+LORA_ALPHA = "lora_alpha"
```
Comment (mmarcinkiewicz) [inline mlperf_logging/mllog/constants.py:170]: I can't really comment in in the right place, but in L40 there are benchmark-specific name constants. Lora should go there as well

File: mlperf_logging/compliance_checker/closed_common.yaml (hunk 2 above)
Comment (itayhubara) [reply mlperf_logging/compliance_checker/closed_common.yaml:1]: correct, I removed it

File: mlperf_logging/compliance_checker/training_4.0.0/closed_llama2_70b_lora.yaml (hunk 4 above)
Comment (itayhubara) [reply mlperf_logging/compliance_checker/training_4.0.0/closed_llama2_70b_lora.yaml:28]: correct I removed the duplication

File: mlperf_logging/mllog/constants.py (hunk 7 above)
Comment (itayhubara) [reply mlperf_logging/mllog/constants.py:170]: in L40 we have benchmark names. Alpha is a hyperparameter. Anyway I added  in L40
LLAMA2_70B_LORA = "llama2_70b_lora"

**Review by nv-rborkar — DISMISSED** [review]

**Comment by nv-rborkar** [comment]
@itayhubara can you please resolve the conflicts so that we can merge. Thanks!

**Comment by itayhubara** [comment]
Done, the root cause of all the issues was GNN PR which was created after llama PR...

**Review by nv-rborkar — APPROVED** [review]

**Pull request merged**